identity-and-access-management/iam/concepts.mdx

```diff
@@ -91,6 +91,14 @@ You are the [Owner](#owner) of the Organization that is created with your Scalew
 
 <Concept>
 
+## Preferred Project
+
+When carrying out actions on Scaleway Object Storage resources via an API or CLI, the API/CLI does not provide a parameter to specify the desired Project for the Object Storage resources. Therefore, when you create an API key with IAM, you are prompted to specify its **preferred Project for Object Storage**. This API key will always use this Project when carrying out Object Storage actions via any API/CLI. See our [dedicated page](/identity-and-access-management/iam/api-cli/using-api-key-object-storage/) for more information.
```

Comment:
Same as dedicated SIS/IAM page: uniformize how to explain the "why" once we agreed on how to say it